src/java.desktop/macosx/native/libawt_lwawt/font/AWTStrike.m

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2014, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2011, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
OpenJDK didn't update the Oracle copyright with the change, not sure if we should be doing it.